grafana-alloy.yaml

```diff
@@ -34,6 +34,11 @@ pipeline:
 
   - uses: strip
 
+  - name: Setup default config files
```

Comment:
You'll need to update the `epoch` so this gets rebuilt with this change.

Reply:
Done thanks! :+1: